packages/client/src/__tests__/integration/happy/count/test.ts

```diff
@@ -41,17 +41,21 @@ test('count', async () => {
   } catch (err) {
     expect(err.message).toMatchInlineSnapshot(`
 
-      Invalid \`prisma.user.aggregate()\` invocation:
+      Invalid \`prisma.user.count()\` invocation:
```

Comment:
Previously undetected bug in the error message, that it used the wrong method name?

Reply:
Indeed, count is an `aggregate` under the hood.